includes/PoolCounter.php

```diff
@@ -161,7 +161,9 @@ function error( $status ) {
 	 * @param $status Status
 	 */
 	function logError( $status ) {
-		wfDebugLog( 'poolcounter', $status->getWikiText() );
+		$msg = json_encode( $status->getErrorsArray() );
+
+		wfDebugLog( 'poolcounter', $msg);
```

Comment:
I'd keep the existing call to `wfDebugLog` and use `WikiaLogger` to log a message with more context - ideally with a context field `errors` with ` $status->getErrorsArray()` (without JSON encoding).